packages/expo-image-picker/android/src/main/java/expo/modules/imagepicker/PickerResultsStore.kt

```diff
@@ -0,0 +1,81 @@
+package expo.modules.imagepicker
+
+import android.content.Context
+import android.content.SharedPreferences
+import android.net.Uri
+import android.os.Bundle
+import android.os.Parcel
+import android.util.Base64
+import java.io.File
+import java.util.*
+import kotlin.collections.ArrayList
+
+const val SHARED_PREFERENCES_NAME = "expo.modules.imagepicker.PickerResultsStore"
+const val EXPIRE_KEY = "expire"
+const val EXPIRATION_TIME = 5 * 60 * 1000 // 5 min
+
+/**
+ * Class that represents a temporary store to promise's results.
+ * It's used if the android kills current activity and we can't resolve promise immediately.
+ */
+class PickerResultsStore(context: Context) {
+  private val sharedPreferences: SharedPreferences =
```

Comment:
What are the exact values that are stored in SharedPreferences?
IIRC, SharedPrefs are not designed to store big amount of data, e.g. bytes from a photo 😞

Reply:
We don't longer save base64 representation into the `SharedPreferences` 😅